hw/formal/data/fpv.mk

```diff
@@ -0,0 +1,44 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+.DEFAULT_GOAL := all
+
+all: build
+
+###################
+## build targets ##
+###################
+build: compile_result
+
+gen_sv_flist:
+	@echo "[make]: gen_sv_flist"
+	cd ${build_dir} && ${sv_flist_gen_cmd} ${sv_flist_gen_opts}
+
+pre_compile: gen_sv_flist
+	@echo "[make]: pre_compile"
+	mkdir -p ${build_dir} && env | sort > ${build_dir}/env_vars
```

Comment:
I think this could just be two commands (since mkdir doesn't have any effect on the output of env).